akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ClientSpec.scala

```diff
@@ -119,6 +121,47 @@ class Http2ClientSpec extends AkkaSpecWithMaterializer("""
         val (_, errorCode) = expectGOAWAY(1)
         errorCode should ===(ErrorCode.COMPRESSION_ERROR)
       }
+      "Three consecutive GET requests" in new SimpleRequestResponseRoundtripSetup {
+        import akka.http.scaladsl.model.headers.CacheDirectives._
+        import headers.`Cache-Control`
+        val requestHeaders = Seq(
+          ":method" -> "GET",
+          ":scheme" -> "https",
+          ":authority" -> "www.example.com",
+          ":path" -> "/",
+          "content-length" -> "0"
+        )
+        requestResponseRoundtrip(
+          streamId = 1,
+          request = HttpRequest(GET, "https://www.example.com/"),
+          expectedHeaders = requestHeaders,
+          response = Seq(
+            HeadersFrame(streamId = 2, endStream = true, endHeaders = true, HPackSpecExamples.C61FirstResponseWithHuffman, None)
```

Comment:
You also annotated it above but what's going on with the streamIds here?!?

Reply:
It also works with the correct stream ids, so could you fix it and create a ticket about looking into why using the wrong ids doesn't break anything?

Reply:
Ah, I guess these tests don't exercise any parts that associate the requests and responses. The HPACK state is also independent for the request and the response, so perhaps it's fine that it didn't cause any problems. Fixed.